### ISSUE TITLE
prxtoll bugs I found

### ISSUE THREAD

**Issue by ogamespec** [pspdev/prxtool#5]
Bug in msub decoding

All msub instructions use $zr instead proper RS register, for example:

   0x0000E5B8: 0x00C1002E '....' - msub       $zr, $at

Should be:

   0x0000E5B8: 0x00C1002E '....' - msub       $a2, $at

Bug in vi2uc.q decoding

0x0000D420: 0xD03C83FE '..<.' - vi2uc.q    R702, C030

Destination is always single register. Following instruction should be decoded as follow:

0x0000D420: 0xD03C83FE '..<.' - vi2uc.q    S723, C030

**Comment by sharkwouter** [comment]
Can anyone confirm if this is still the case?

**Comment by artart78** [comment]
Yup, I checked and it's still the case. Also, psp-binutils returns the correct result, and ghidra-allegrex too.
It shouldn't be too hard to fix, but prxtool is mostly useless nowadays anyway since we have ghidra-allegrex.
